ckanext/unaids/blueprints/login_register_catch.py

```diff
@@ -0,0 +1,25 @@
+import logging
+import ckan.plugins.toolkit as toolkit
+from flask import Blueprint
+
+log = logging.getLogger(__name__)
+
+login_register_catch = Blueprint("login_register_catch", __name__)
+
+
+def disable_default_login():
+    """
+    Override and redirect default Login route
+    """
+    return toolkit.redirect_to("/user/saml2login")
+
+
+def disable_default_register():
```

Comment:
I think that right now it's best to leave only one function here used for both blueprint rules.